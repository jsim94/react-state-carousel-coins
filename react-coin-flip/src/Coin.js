import React from "react";
import "./Coin.css";

/**
 *
 * @param {boolean} side true - heads, false - tails
 */

function Coin({ side }) {
  return <div className="Coin" data-testid="coin" style={{ background: side ? "green" : "red" }}></div>;
}

export default Coin;
