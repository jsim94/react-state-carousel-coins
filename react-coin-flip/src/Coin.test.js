import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Coin from "./Coin";

it("should render", function () {
  render(<Coin />);
});

it("should match snapshot", function () {
  const { asFragment } = render(<Coin />);
  expect(asFragment()).toMatchSnapshot();
});
