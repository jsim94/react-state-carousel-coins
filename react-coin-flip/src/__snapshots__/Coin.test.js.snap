// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Coin"
    data-testid="coin"
    style="background: red;"
  />
</DocumentFragment>
`;
